ibm_catalog.json

```diff
@@ -21,20 +21,21 @@
         "ai"
       ],
       "short_description": "Creates and configures an instance of IBM watsonx.data.",
-      "long_description": "This architecture supports creating and configuring an instance of watsonx.data.",
+      "long_description": "This architecture supports creating and configuring an instance of watsonx.data. For more details on watsonx data, [see here](https://cloud.ibm.com/docs/watsonxdata/index.html).",
       "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/blob/main/README.md",
       "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-data/main/images/watsonx-data.svg",
       "provider_name": "IBM",
       "features": [
         {
           "title": "Creates an instance of IBM watsonx.data",
-          "description": "Creates and configures an IBM watsonx.data instance."
+          "description": "For more details on Watsonx Data instances, [see here](https://cloud.ibm.com/watsonxdata#about)"
         },
         {
           "title": "Creates a KMS encryption enabled IBM watsonx.data instance",
           "description": "Create and configures a KMS encryption enabled IBM watsonx.data instance for Enterprise plan."
         }
       ],
+      "support_details": "This product is in the community registry, as such support is handled through the originated repo. If you experience issues please open an issue in that repository [https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/issues](https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/issues). Please note this product is not supported via the IBM Cloud Support Center.",
```

Comment:
Do not need to show the full link (already discussed in DA rallies). Please hyperlink. Refer [this](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc/blob/issue_13328/ibm_catalog.json#L74)